src/docs/doxygen/GeosxConfig.hpp

```diff
@@ -11,13 +11,15 @@
 /// GEOSX major version number
 #define GEOSX_VERSION_MAJOR 0
 /// GEOSX minor version number
-#define GEOSX_VERSION_MINOR 0
+#define GEOSX_VERSION_MINOR 2
 /// GEOSX patch version number
-#define GEOSX_VERSION_PATCH 31
+#define GEOSX_VERSION_PATCH 0
 /// GEOSX full version number string
-#define GEOSX_VERSION_FULL  "0.0.31"
+#define GEOSX_VERSION_FULL  "0.2.0"
+/// GEOSX full version number string
+#define GEOSX_VERSION_DEV  "feature/klevzoff/version-bccb96b9f"
```

Comment:
Oops, that is definitely a problem. Will add a workaround.

Reply:
After you merge, just run on `develop`, then checkout a new branch to merge into develop....but it isn't really a big deal since this isn't really used.